homeworks/yehor.honcharenko_NOcode-developer/GamesProject/style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  width: 1280px;
+  margin: 0 auto;
+  overflow-x: hidden;
+}
+
+* {
+  font-family: Roboto, sans-serif;
+}
+
+/*
+.header {
+    background: #EF4934;
+    width: 100vw;
+    height: 80px;
+}
+*/
+
+.about-me {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: space-between;
+  margin: 53px 52px 0 53px;
+  gap: 68px;
+}
+
+.img-about {
+  width: 50%;
+  max-width: none;
+  display: block;
+  margin: 0 auto;
+}
+
+img {
+  width: 100%;
+}
+
+.text-about {
+  width: 50%;
+  max-width: none;
+  display: block;
+}
+
+.text-about>*+* {
+  margin-top: 24px;
+}
+
+h1 {
+  color: #221F1F;
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 32px;
+  margin: 0;
+}
+
+p {
+  color: #221F1F;
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 32px;
+  margin: 0;
+}
+
+a {
+  color: #EF4934;
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 32px;
+  text-decoration-line: underline;
```

Comment:
All links are underlined by default, so you can skip this.
<img width="1037" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/ae78d133-49e1-4885-b663-040b6799cb4c">

Reply:
removed